File: StackSearch/StackSearch/Models/Items/Owner.swift
//
//  Owner.swift
//  StackSearch
//
//  Created by Saiful I. Wicaksana on 11/08/18.
//  Copyright © 2018 icaksama. All rights reserved.
//

import Foundation

internal struct Owner: Codable {
    internal let reputation: Int?
    internal let user_id: Int?
    internal let user_type: String
    internal let profile_image: String?
    internal let display_name: String?
    internal let link: String?
}

File: AStackSearch/app/src/main/java/com/icaksama/astacksearch/models/items/Owner.java
package com.icaksama.astacksearch.models.items;

import com.google.gson.annotations.SerializedName;

/**
 * Created by icaksama on 13/08/18.
 */

public class Owner {

    @SerializedName("reputation")
    public int reputation;
    @SerializedName("user_id")
    public int user_id;
    @SerializedName("user_type")
    public String user_type;
    @SerializedName("profile_image")
    public String profile_image;
    @SerializedName("display_name")
    public String display_name;
    @SerializedName("link")
    public String link;
}
